### PR TITLE
ER-1119: Video bundle paragraphs type

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/ER-1119

Adds the Video bundle paragraphs type.

### CODE REVIEW

File: features/breol_video/breol_video.features.field_instance.inc [hunk 1]
```diff
@@ -109,8 +109,215 @@ function breol_video_field_default_field_instances() {
...
+      'paragraphs_editor_preview' => array(
+        'label' => 'above',
+        'settings' => array(),
+        'type' => 'hidden',
+        'weight' => 0,
+      ),
+    ),
+    'entity_type' => 'paragraphs_item',
+    'field_name' => 'field_material_carousel_color',
+    'label' => 'Farve',
```
Comment (cableman) [inline features/breol_video/breol_video.features.field_instance.inc:176]: Danish label?

File: features/breol_video/breol_video.features.inc [hunk 2]
```diff
@@ -33,3 +33,17 @@ function breol_video_node_info() {
...
   return $items;
 }
+
+/**
+ * Implements hook_paragraphs_info().
+ */
+function breol_video_paragraphs_info() {
+  $items = array(
+    'video_bundle' => array(
+      'name' => 'Videobundt',
```
Comment (cableman) [inline features/breol_video/breol_video.features.inc:43]: "Videobundt" should be wrapped in t() function.

File: features/breol_video/breol_video.info [hunk 3]
```diff
@@ -4,18 +4,31 @@ core = 7.x
...
 version = 7.x-1.1
 project = breol_video
+dependencies[] = breol_carousel_paragraph
 dependencies[] = ctools
+dependencies[] = entityreference
 dependencies[] = features
 dependencies[] = file_entity
 dependencies[] = image
 dependencies[] = media
-dependencies[] = node
```
Comment (cableman) [inline features/breol_video/breol_video.info:12]: Removed dependency on node?

File: features/breol_video/breol_video.module [hunk 4]
```diff
@@ -74,3 +74,70 @@ function breol_frontend_view_video($node) {
...
+ *
+ * @param string $hex_color
+ *   The color as a hex string (#rrggbb).
+ *
+ * @return string
+ *   The contrast color as a hex string (#rrggbb).
+ *
+ * @see https://stackoverflow.com/a/42921358
+ */
+function _breol_video_get_contrast_color($hex_color) {
```
Comment (cableman) [inline features/breol_video/breol_video.module:106]: Very magical function :-) thanks for the overflow link 👍

File: features/breol_video/breol_video.features.field_instance.inc (hunk 1 above)
Comment (rimi-itk) [reply features/breol_video/breol_video.features.field_instance.inc]: Nice catch. Fixed.

**Pull request merged**